simd/src/Kokkos_SIMD_AVX512.hpp

```diff
@@ -519,11 +519,11 @@ class simd<std::uint64_t, simd_abi::avx512_fixed_size<8>> {
   }
   KOKKOS_IMPL_HOST_FORCEINLINE_FUNCTION void copy_from(value_type const* ptr,
                                                        element_aligned_tag) {
-    m_value = _mm512_loadu_epi64(ptr);
+    m_value = _mm512_loadu_si512(ptr);
   }
   KOKKOS_IMPL_HOST_FORCEINLINE_FUNCTION void copy_to(
       value_type* ptr, element_aligned_tag) const {
-    _mm512_storeu_epi64(ptr, m_value);
+    _mm512_storeu_si512(ptr, m_value);
```

Comment:
Why are these showing in the diff?  These changes were in #6183

Reply:
Should disappear after rebasing now.